docker/latest/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM julia:1.3.1-buster
```

Comment:
Any reason for `docker/latest/Dockerfile` instead of `docker/Dockerfile`?

Reply:
the subfolder is recognized as the tag of the docker image, for this image, the tag will be `latest`, which is the default one when `docker pull turinglang/turing` is used.

We can provide other folders for different versions of Turing, like `0.8.4`, `0.8.5`, if our docker file is accepted by the official docker image repo, these tags will appear on the main page of the image, like this:
https://hub.docker.com/_/perl, the section "Supported tags and respective Dockerfile links".